packages/wrangler/src/__tests__/r2.test.ts

```diff
@@ -42,11 +42,12 @@ describe("r2", () => {
 			Manage R2 buckets
 
 			Commands:
-			  wrangler r2 bucket create <name>  Create a new R2 bucket
-			  wrangler r2 bucket list           List R2 buckets
-			  wrangler r2 bucket delete <name>  Delete an R2 bucket
-			  wrangler r2 bucket sippy          Manage Sippy incremental migration on an R2 bucket
-			  wrangler r2 bucket notification   Manage event notifications for an R2 bucket
+			  wrangler r2 bucket create <name>             Create a new R2 bucket
+			  wrangler r2 bucket put-storage-class <name>  Update the default storage class of an existing R2 bucket
```

Comment:
Have you considered making a `put` subcommand so we can perhaps nest other bucket updates under it?
`wrangler r2 bucket put storage-class <name>`

Reply:
Good idea -- we indeed should.

Reply:
Added as a subcommand.